spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -239,4 +239,126 @@
       end
     end
   end
+
+  describe '#call' do
+    let(:pii_from_user) { flow.flow_session[:pii_from_user] }
+    let(:params) { ActionController::Parameters.new({ state_id: submitted_values }) }
+    let(:capture_secondary_id_enabled) { true }
+    let(:dob) { '1972-02-23' }
+    let(:same_address_as_id) { 'false' } # value on submission
+    # residential
+    let(:address1) { '123 Sesame Street' }
+    let(:address2) { 'Apt. #C' }
+    let(:city) { 'Twin Peaks' }
+    let(:state) { 'Nevada' }
+    let(:zipcode) { '90001' }
+    # identity_doc_
+    let(:identity_doc_address1) { '123 Sesame Street' }
+    let(:identity_doc_address2) { 'Apt. #C' }
+    let(:identity_doc_city) { 'Twin Peaks' }
+    let(:identity_doc_address_state) { 'Nevada' }
+    let(:identity_doc_zipcode) { '90001' }
+
+    let(:submitted_values) do
+      {
+        dob:,
+        same_address_as_id:,
+        address1:,
+        address2:,
+        city:,
+        state:,
+        zipcode:,
+        identity_doc_address1:,
+        identity_doc_address2:,
+        identity_doc_city:,
+        identity_doc_address_state:,
+        identity_doc_zipcode:,
+      }
+    end
+
+    before(:each) do
+      allow(IdentityConfig.store).to receive(:in_person_capture_secondary_id_enabled).
+        and_return(true)
+      allow(user).to receive(:establishing_in_person_enrollment).
+        and_return(enrollment)
+    end
+
+    # User picks "Yes, I live at the address on my state-issued ID"
```

Comment:
can we break these comments up and put them in the test at the point were we're doing each behavior described?